test/pet.test.js

```diff
@@ -0,0 +1,189 @@
+const Pet = require('../src/pet');
+
+let testPet;
+
+const deadMessage = "Your pet is no longer alive :("
+
+beforeEach(() => {
+    testPet = new Pet();
+  });
+
+describe('constructor', () => {
+    it('should return a new pet with a name', () => {
+        expect(new Pet()).toBeInstanceOf(Object);
+        expect(new Pet().name).toBe("An unloved pet");
+        expect(new Pet('Manny').name).toBe("Manny");
+    })
+
+    it('should return a new pet with an age', () => {
+        expect(new Pet().age).toBe(0);
+    })
+
+    it('shold return a new pet with a hunger', () => {
+        expect(new Pet().hunger).toBe(0);
```

Comment:
Same as above, could change the test name to match exactly what your test is checking.